src/triage/component/results_schema/alembic/versions/264786a9fe85_add_label_value_to_prodcution_table.py

```diff
@@ -0,0 +1,54 @@
+"""add label_value to prodcution table
+
+Revision ID: 264786a9fe85
+Revises: 1b990cbc04e4
+Create Date: 2019-02-26 13:17:05.365654
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '264786a9fe85'
+down_revision = '1b990cbc04e4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_table("list_predictions", schema="production")
+    op.create_table(
+        "list_predictions",
+        sa.Column("model_id", sa.Integer(), nullable=False),
+        sa.Column("entity_id", sa.BigInteger(), nullable=False),
+        sa.Column("as_of_date", sa.DateTime(), nullable=False),
+        sa.Column("score", sa.Numeric(), nullable=True),
+        sa.Column('label_value', sa.Integer, nullable=True),
+        sa.Column("rank_abs", sa.Integer(), nullable=True),
+        sa.Column("rank_pct", sa.Float(), nullable=True),
+        sa.Column("matrix_uuid", sa.Text(), nullable=True),
+        sa.Column("test_label_window", sa.Interval(), nullable=True),
+        sa.ForeignKeyConstraint(["model_id"], ["model_metadata.models.model_id"]),
+        sa.PrimaryKeyConstraint("model_id", "entity_id", "as_of_date"),
+        schema="production",
+    )
+
+
+def downgrade():
+    op.drop_table("list_predictions", schema="production")
+    op.create_table(
+        "list_predictions",
+        sa.Column("model_id", sa.Integer(), nullable=False),
+        sa.Column("entity_id", sa.BigInteger(), nullable=False),
+        sa.Column("as_of_date", sa.DateTime(), nullable=False),
+        sa.Column("score", sa.Numeric(), nullable=True),
+        sa.Column("rank_abs", sa.Integer(), nullable=True),
+        sa.Column("rank_pct", sa.Float(), nullable=True),
+        sa.Column("matrix_uuid", sa.Text(), nullable=True),
+        sa.Column("test_label_window", sa.Interval(), nullable=True),
+        sa.ForeignKeyConstraint(["model_id"], ["results.models.model_id"]),
```

Comment:
This should be model_metadata (Alena hit this error)

Reply:
@tweddielin

Reply:
I actually ended up doing this when I was fixing conflicts (I had to fix the Alembic revisions to get the tests working after conflict fixing)